.circleci/config.yml

```diff
@@ -45,6 +45,7 @@ jobs:
     working_directory: /opt/app-root/apicast
     steps:
       - checkout
+      - run: rm -rf lua_modules
```

Comment:
Why is this needed now, but not before? :thinking:

Reply:
It is needed for local run. There is no way to ignore folders on local build.